### DISCUSSION TITLE
I got approved but still: Are you a student?

### DISCUSSION THREAD

**Discussion by dhqcz** [community/community#70493] [GitHub Education]
### Hi everyone,

Hello,
I have been Approved on Oct 1, 2023.
**Congrats! Your request for @dhqcz was approved and your benefits are now available.**

But anytime I try to use a benefit for student like Student Dev Pack, it says: **Are you a student?**
I click **Yes** and it says: **Enjoy the benefits of being a student Hi, @dhqcz! You were last verified as a student on Oct 1, 2023.**
Then I choose: **GitHub Student Developer Pack** and I am back to the **Are you a student?** page!

When I try to verify again, it says: ****Hi, dhqcz! You were last verified as a student on . It is not necessary for you to re verify at this time.**

So please what is going on. I have patiently waited for 18 days and nothing changed!

**Comment by zye212** [comment]
Same problem here. 
And in https://education.github.com/discount_requests/application it shows that I have been approved, but I am still unable to access any of the student benefits. 
![image](https://github.com/community/community/assets/91609168/c1134fb1-0124-4e58-a9c1-405b05716d7b)
I also didn't get any email confirming my approvl status.

**Comment by XenonyxBlaze** [comment]
Same problem here.

**Comment by ghost** [comment]
Remeber when wrong credentials might lead to the delay.
